justfile

```diff
@@ -192,6 +195,14 @@ k3d-create: && _k3d-ready
         --kubeconfig-update-default \
         --kubeconfig-switch-context=false
 
+# Creates two k3d clusters with a shared network, that can be used for multicluster testing.
+k3d-create-multicluster: && _k3d-ready
+	k3d cluster create {{ k3d-name }} \
+		--image='+{{ k3d-k8s }}' \
+		--agents='{{ k3d-agents }}' \
+		--servers='{{ k3d-servers }}' \
+		--network='{{ k3d-network }}'
+
```

Comment:
I don't think this is correct:

> Creates **two** k3d clusters

The workflow itself is still responsible for creating clusters with specific names... I think we should figure out how to get this unified.